internal/dwarf/frame/table.go

```diff
@@ -44,65 +47,64 @@ type InstructionContext struct {
 	RetAddrReg    uint64
 	codeAlignment uint64
 	dataAlignment int64
+	ArgsSize      uint64
 }
 
 func (instructionContext *InstructionContext) Loc() uint64 {
 	return instructionContext.loc
 }
 
-// Represents a whole native function.
-type FrameContext struct {
+// Context represents a whole native function.
+type Context struct {
```

Comment:
nit: I find the new name a bit less descriptive

Reply:
I understand that/ We need to think it in the context. It is an idiomatic Go thing; the name of the package is already `frame`.